package.json

```diff
@@ -61,7 +62,7 @@
     "watch": "tsc -w",
     "prepublish": "npm run clean && npm run compile && npm run build",
     "test": "TS_NODE_PROJECT=src/tsconfig.json nyc mocha --exit",
-    "lint": "tslint -p ./",
+    "lint": "eslint src/**/*.ts",
```

Comment:
I believe this will exclude the test directory. You probably want `eslint **/*.ts`.